ogusa_calibrate/psid_data_setup.py

```diff
@@ -7,22 +7,23 @@
 from linearmodels import PanelOLS
 from rpy2.robjects import r
 from rpy2.robjects import pandas2ri
+
 pandas2ri.activate()
-pd.options.mode.chained_assignment = 'raise'
+pd.options.mode.chained_assignment = "raise"
 
-'''
+"""
 This script takes PSID data created from psid_download.R and:
 1) Creates variables at the "tax filing unit" (equal to family unit in
    PSID since there is no info on the filing status chosen).
 2) Selects a sample of observations to work with (e.g., dropping very
    old, very low income, etc.).
 3) Computes a measure of lifetime income and places each household into
    a lifetime income percentile group
-'''
+"""
```

Comment:
Are the double quotes everywhere a `black` thing @MaxGhenis?  You're killing me with these :-)

Reply:
Yep will definitely make these separate commits in the future!